### PR TITLE
[AC-1512] Feature: Secrets Manager billing - round 2

### PR DESCRIPTION
**Pull request by eliykat**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Code changes

- **file.ext:** Description of what was changed and why

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by differsthecat — APPROVED** [review]
This looks good for Vault, the only change I see for us is a service name change in SyncService.

File: libs/common/src/platform/misc/utils.ts [hunk 1]
```diff
@@ -538,6 +538,17 @@ export class Utils {
...
   }
 
+  /**
+   * Return the number of days remaining before a target date arrives.
+   * Returns 0 if the day has already passed.
+   */
+  static daysRemaining(targetDate: Date): number {
+    const diffTime = targetDate.getTime() - Date.now();
+    const [ms, seconds, minutes, hours] = [1000, 60, 60, 24];
+    const msPerDay = ms * seconds * minutes * hours;
```
Comment (MGibson1) [inline libs/common/src/platform/misc/utils.ts:548]: Why allocate and calculate this every time the method is called? It's a constant (86400000)
```suggestion
    const msPerDay = 86400000;
```

File: libs/common/src/platform/misc/utils.ts [hunk 2]
```diff
@@ -538,6 +538,17 @@ export class Utils {
...
+    return Math.max(0, Math.round(diffTime / msPerDay));
```
Comment (MGibson1) [inline libs/common/src/platform/misc/utils.ts:549]: likely this should be a ceiling. I'm not sure how the deactivation will work, but it probably doesn't use a round to determine if we should deactivate an account 😉

**Review by MGibson1 — CHANGES_REQUESTED** [review]
These are relatively small issues. Should be quick to fix, but feel free to barrel over my review if needed.

File: libs/common/src/platform/misc/utils.ts (hunk 2 above)
Comment (MGibson1) [reply libs/common/src/platform/misc/utils.ts]: yup, I had the direction wrong in my head! thanks

**Review by MGibson1 — APPROVED** [review]

File: apps/web/src/app/billing/organizations/secrets-manager/sm-subscribe.component.html [hunk 3]
```diff
@@ -56,10 +56,13 @@ <h4 bitTypography="h4">{{ "secretsManagerForPlan" | i18n : planName }}</h4>
...
           <input bitInput formControlName="additionalServiceAccounts" type="number" />
-          <bit-hint>{{
-            "additionalServiceAccountsDesc"
-              | i18n : serviceAccountsIncluded : (monthlyCostPerServiceAccount | currency : "$")
-          }}</bit-hint>
+          <bit-hint>
+            {{ "includedServiceAccounts" | i18n : serviceAccountsIncluded }}
+            {{
+              "addAdditionalServiceAccounts"
+                | i18n : (monthlyCostPerServiceAccount | currency : "$")
```
Comment (cyprain-okeke) [inline apps/web/src/app/billing/organizations/secrets-manager/sm-subscribe.component.html:63]: change monthlyCostPerServiceAccount to monthlyCostPerServiceAccounts

Comment (r-tome) [reply apps/web/src/app/billing/organizations/secrets-manager/sm-subscribe.component.html:63]: Its the value for a singular service account

**Review by r-tome — APPROVED** [review]
LGTM

**Review by cyprain-okeke — APPROVED** [review]
it looks great

**Review by differsthecat — APPROVED** [review]

**Pull request merged**